util/grunt/.jshintrc

```diff
@@ -19,7 +19,8 @@
     "Env": true,
     "Login": true,
     "moment": true,
-    "Config": true
+    "Config": true,
+    "Newgame": true
```

Comment:
Note to self: this seems weird.  I assume you're importing Newgame variables into the Tourn JS somehow, but that's not how we treat anything else in the API except stuff that really *is* global, like, everything else which actually belongs to a single page is kept in its namespace.  So i want to look into this more later.

Reply:
^ I realised we never looped back to this, and i don't think it's right.  At the very minimum, can you explain in words here in the PR why it's correct to add "Newgame" to jshintrc, and we can take it from there?

Reply:
I'm going to try removing it and then running unit tests to see why I had it there in the first place.